conduct.md

```diff
@@ -213,6 +224,14 @@ Any decision made by the instructor for the given infraction(s) will be final.
 
 Janyl Jumadinova
 :heavy_exclamation_mark: TODO: Add your name
+
+=======
 Bennett Westfall
 Zackery Devers
 Natnael Kahesay
+Kobe Coleman
+Adriana Solis
+Favour Ojo
+Kai'lani Woodward
+Zoe White
```

Comment:
Formatting the names like this will put them all on the same line once Markdown is rendered, It would be great to add commas between names to make this section more readable.